src/rospkg/os_detect.py

```diff
@@ -713,15 +713,16 @@ def get_codename(self):
 OS_NEON = 'neon'
 OS_OPENEMBEDDED = 'openembedded'
 OS_OPENSUSE = 'opensuse'
-OS_TIZEN = 'tizen'
 OS_OPENSUSE13 = 'opensuse'
+OS_TIZEN = 'tizen'
```

Comment:
Please revert unrelated changes like this.

Reply:
I moved those changes to a separate commit. I can remove them if you want, but it just annoyed me that the definitions were in a different order than the registrations.